test/test_kss_builder_base_twig.js

```diff
@@ -296,7 +296,7 @@ describe('KssBuilderBaseTwig object API', function() {
         source: helperUtils.fixtures('source-twig-builder-test'),
         destination: path.resolve(__dirname, 'output', 'base_twig', 'buildPage'),
         builder: helperUtils.fixtures('builder-twig-with-assets'),
-        homepage: 'alternate-homepage.md'
+        homepage: helperUtils.fixtures('source-twig-builder-test') + '/alternate-homepage.md'
```

Comment:
Making use of `path.join()` could save some potential issues on source not being found.

Reply:
I see `helperUtils.fixtures` use `path.join` under the hood.